### ISSUE TITLE
AlphaX: eXploring Neural Architectures with Deep Neural Networks and Monte Carlo Tree Search

### ISSUE THREAD

**Issue by l1t1** [leela-zero/leela-zero#2339]
https://arxiv.org/abs/1903.11059
  We present AlphaX, a fully automated agent that designs complex neural architectures from scratch. AlphaX explores the exponentially grown search space with a distributed Monte Carlo Tree Search (MCTS) and a Meta-Deep Neural Network (DNN). MCTS intrinsically improves the search efficiency by dynamically balancing the exploration and exploitation at fine-grained states, while Meta-DNN predicts the network accuracy to guide the search, and to provide an estimated reward to speed up the rollout. As the search progresses, AlphaX also generates the training data for Meta-DNN. So, the learning of Meta-DNN is end-to-end. In 14 days with only 16 GPUs (1832 samples), AlphaX found an architecture that reaches the state-of-the-art accuracies on both CIFAR-10(97.18%) and ImageNet(75.5% top-1 and 92.2% top-5). This demonstrates up to 10x speedup over the original searching for NASNet that used 500 GPUs in 4 days (20000 samples). On NASBench-101, AlphaX demonstrates 3x and 2.8x speedup over Random Search and Regularized Evolution. Finally, we show the searched architecture improves a variety of vision applications from Neural Style Transfer, to Image Captioning and Object Detection. Our implementation is available at https://github.com/linnanwang/AlphaX-NASBench101

**Comment by alreadydone** [comment]
Strangely there's an earlier version (updated in February) https://arxiv.org/abs/1805.07440 that doesn't have Yuandong Tian as an author...

**Comment by Ttl** [comment]
Doesn't seem relevant for LZ.

**Comment by alreadydone** [comment]
Another recent paper on NAS: https://arxiv.org/abs/1904.02877
For comments see https://jack-clark.net/2019/04/15/import-ai-142-import-ai-142-berkeley-spawns-cheap-blue-arm-google-trains-neural-nets-to-prove-math-theorems-seven-questions-about-gans/

**Comment by l1t1** [comment]
MorphNet: Fast & Simple Resource-Constrained Structure Learning of Deep
https://zhuanlan.zhihu.com/p/62888707
https://arxiv.org/pdf/1711.06798.pdf
https://github.com/google-research/morph-net

**Comment by l1t1** [comment]
https://zhuanlan.zhihu.com/p/62865145
test TensorFlow, PyTorch adnd MXNet on Titan RTX

**Comment by l1t1** [comment]
https://arxiv.org/pdf/1904.07392.pdf

**Comment by l1t1** [comment]
https://arxiv.org/search/?query=Linnan+Wang&searchtype=author

**Comment by l1t1** [comment]
https://arxiv.org/abs/1904.09274
